### ISSUE TITLE
Feature: Login with GitHub

### ISSUE THREAD

**Issue by robotdan** [FusionAuth/fusionauth-issues#33]
### Problem
The problem is I can't login to FusionAuth with GitHub. :-) 

### Solution
Add the ability to login with GitHub.

### Alternatives/workarounds
None

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by arunkumar413** [comment]
Yes, login with github is very important for developer community. Please prioritize this

**Comment by robotdan** [comment]
In version 1.6.0, you should be able to use the OpenID Connect IdP in FusionAuth to login with GitHub. Their impl is not OpenId Connect, but it will work, we'll be documenting how to configure this shortly. 

https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/

The GitHub user will have to have their email public for this to work.

**Comment by jobannon** [comment]
This was delivered (in practice) through OIDC.  
Related Documentation can be found here: https://fusionauth.io/docs/v1/tech/identity-providers/openid-connect/github/

**Comment by mooreds** [comment]
Closing because as @jobannon says, we support and document this.
